nodeup/pkg/model/kube_apiserver.go

```diff
@@ -94,10 +94,10 @@ func (b *KubeAPIServerBuilder) Build(c *fi.ModelBuilderContext) error {
 	// @check if we are using secure client certificates for kubelet and grab the certificates
 	if b.UseSecureKubelet() {
 		name := "kubelet-api"
-		if err := buildCertificateRequest(c, b.NodeupModelContext, name, ""); err != nil {
+		if err := b.BuildCertificateTask(c, name, name+".pem"); err != nil {
```

Comment:
adding quick link for easy review https://github.com/kubernetes/kops/blob/master/nodeup/pkg/model/context.go#L327 .. args for the both methods is context, name and filename